fc_karpaty/src/main/java/tytarchuk/Table.java

```diff
@@ -0,0 +1,33 @@
+package tytarchuk;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Table {
+    private final SelenideElement table;
+
+    public Table(SelenideElement table) {
+        this.table = table;
+    }
+
+    public List<SelenideElement> getLines() {
+        ElementsCollection lines = table.$$x(".//tr");
```

Comment:
What about to specify xpath to avoid removing zero element

Reply:
done